src/main/java/kdt/springbootboardjpa/global/exception/ExceptionResolver.java

```diff
@@ -0,0 +1,20 @@
+package kdt.springbootboardjpa.global.exception;
+
+import jakarta.servlet.http.HttpServletRequest;
+import kdt.springbootboardjpa.global.dto.ApiError;
+import kdt.springbootboardjpa.global.dto.BaseResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class ExceptionResolver {
+
+    @ResponseStatus(HttpStatus.NOT_FOUND)
+    @ExceptionHandler({IllegalArgumentException.class})
+    public BaseResponse handleForbiddenException(HttpServletRequest request, RuntimeException e) {
```

Comment:
```suggestion
    public BaseResponse handleForbiddenException(HttpServletRequest request, RuntimeException e) {
```

컴파일러에게 타입 추론을 맡겨야 하는 상황이 아니면 제네릭 타입을 명시해주는 것이 더 좋습니다.
더 좋은 이유는 컴파일러가 제네릭 Raw 타입을 어떻게 타입 추론하는지에 대해서 알아보면 알 수 있긴합니다만,
여기서는 일단 제네릭 타입을 명시하는 것이 좋다 정도만 알고 넘어가면 좋지 않을까 싶네요.!